### ISSUE TITLE
error of training

### ISSUE THREAD

**Issue by yiColddrink** [clovaai/deep-text-recognition-benchmark#53]
when i use ST_spe and validation from https://drive.google.com/drive/folders/192UfE9agQUMNq6AgU3_E05_FcPZK4hyt
```
CUDA_VISIBLE_DEVICES=2 python3 train.py --train_data data/ST_spe --valid_data data/validation --select_data / --batch_ratio 1  --Transformation TPS --sensitive  --workers 0  --FeatureExtraction ResNet --SequenceModeling BiLSTM --Prediction Attn
```
The training seems to be stuck

```
[0/300000] Loss: 4.57599 elapsed_time: 3.90644
TTkkkkkkk//////////////, gt: SANDS               ,   False
T7777777777777777777777, gt: PULL                ,   False
TTTTeeeeTeTeTeTeTeTeTeT, gt: DIRECTION           ,   False
TTTTTTTTTTTTTTTTTTTTTTT, gt: ALTERNATE           ,   False
TT77`7`7`777777,777,,77, gt: FOR                 ,   False
[0/300000] valid loss: 4.65960 accuracy: 0.000, norm_ED: 36773.78
best_accuracy: 0.000, best_norm_ED: 36773.78
```

log:
```
------------ Options -------------
experiment_name: TPS-ResNet-BiLSTM-Attn-Seed1111
train_data: data/ST_spe
valid_data: data/validation
manualSeed: 1111
workers: 0
batch_size: 64
num_iter: 300000
valInterval: 2000
continue_model: 
adam: False
lr: 1
beta1: 0.9
rho: 0.95
eps: 1e-08
grad_clip: 5
select_data: ['/']
batch_ratio: ['1']
total_data_usage_ratio: 1.0
batch_max_length: 23
imgH: 32
imgW: 280
rgb: False
character: 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
sensitive: True
PAD: False
data_filtering_off: False
Transformation: TPS
FeatureExtraction: ResNet
SequenceModeling: BiLSTM
Prediction: Attn
num_fiducial: 20
input_channel: 1
output_channel: 512
hidden_size: 256
num_gpu: 1
num_class: 96
---------------------------------------
```

Thank you for your reply.

**Comment by yiColddrink** [comment]
Solved, Just wait for a moment

**Comment by little2Rabbit** [comment]
@xkyi Training got stuck. How did you work it out，there are generated models, but the identification is poor

**Comment by little2Rabbit** [comment]
@xkyi 

Training information
[1/300] Train loss: 2.46951, Valid loss: 2.46787, Elapsed_time: 1.73494
Current_accuracy : 0.000, Current_norm_ED  : 0.01
Best_accuracy    : 0.000, Best_norm_ED     : 0.01
--------------------------------------------------------------------------------
Ground Truth              | Prediction                | Confidence Score & T/F
--------------------------------------------------------------------------------
350                       |                           | 0.0000  False
2209                      | 9999999999999999999999999 | 0.0000  False
1983                      |                           | 0.0000  False
0130                      | 9999999999999999999999999 | 0.0000  False
--------------------------------------------------------------------------------
end the training


test information
model input parameters 32 100 20 1 512 256 12 25 TPS ResNet BiLSTM Attn
loading pretrained model from saved_models/TPS-ResNet-BiLSTM-Attn-Seed1111/best_accuracy.pth
dataset_root:    test    dataset: /
sub-directory:  /.       num samples: 14
0.000

**Comment by xkyi** [comment]
> @xkyi
> 
> ## Training information
> [1/300] Train loss: 2.46951, Valid loss: 2.46787, Elapsed_time: 1.73494
> Current_accuracy : 0.000, Current_norm_ED : 0.01
> Best_accuracy : 0.000, Best_norm_ED : 0.01
> ## Ground Truth | Prediction | Confidence Score & T/F
> ## 350 | | 0.0000 False
> 2209 | 9999999999999999999999999 | 0.0000 False
> 1983 | | 0.0000 False
> 0130 | 9999999999999999999999999 | 0.0000 False
> end the training
> 
> test information
> model input parameters 32 100 20 1 512 256 12 25 TPS ResNet BiLSTM Attn
> loading pretrained model from saved_models/TPS-ResNet-BiLSTM-Attn-Seed1111/best_accuracy.pth
> dataset_root: test dataset: /
> sub-directory: /. num samples: 14
> 0.000

the training is slow

**Comment by little2Rabbit** [comment]
@xkyi But only show once, wait a long time，print end the training
